docs/source/recipes.rst

```diff
@@ -437,3 +437,38 @@ different environment.
 
    To test operations with a small interactive job, a 'test' group can be used
    to ensure that the operations do not try to run on multiple cores or GPUs.
+
+Pass options to operations run through containers or another environment
+========================================================================
+
+When executing an operation in a container (e.g. Singularity or Docker) or a different environment
+``FlowGroups`` can be used to passed options to an ``exec`` command. Below is an example which shows
+how to use this feature to get debug information from an operation executed in a container.
+
+.. code-block:: python
+
+    # project.py
+    from flow import FlowProject
+
+
+    class Project(flow.FlowProject):
+        pass
+
+
+    debug = Project.make_group("debug", run_options="--debug")
```

Comment:
Will _anything_ in `run_options` be passed to the exec command? If so, add a comment at the end of the line

Reply:
What do you mean by anything?